webpack.config.js

```diff
@@ -102,16 +102,7 @@ const config = {
       {
         test: /\.scss$/,
         include: stylesDir,
-        exclude: stylesDir + "/embed/index.scss",
```

Comment:
Now we just inject the styles into the page w/ javascript like all of our other styles